CircularProgress.xcodeproj/project.pbxproj

```diff
@@ -85,6 +87,14 @@
 			path = Example;
 			sourceTree = "<group>";
 		};
+		F1FDA5872254CE7D00797872 /* protocol */ = {
+			isa = PBXGroup;
+			children = (
+				F1FDA5882254CE8D00797872 /* MouseTrackable.swift */,
+			);
+			path = protocol;
```

Comment:
I prefer to not nest it into directories.

Reply:
ok. Flat hierarchy it is.

Reply:
And a comment on MouseTrackable being more independent. Yeah. Id love that too, but no way around overriding hitTesting inside the class it applies to etc. I have tried many times in the past. This latest stab for Hybrid and CircularProgress I feel is a nice compromise. I also got rid of the storing of NSTracking area in the class, now we just derived them from Apples API

Reply:
Skipping this. As I think this is as deep as the rabbit hole goes on this. I could be wrong though. But maybe leave for 🔮 improvement? I feel a refactoring and cleaning up of code would be more appropriate before this tweak 🤷